lib/workflows/workflow.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/**
+ * Workflow data entry structure
+ *
+ * @returns {object} - content structure to match requirements of punchcard-content-types
+ */
+module.exports = {
+  name: 'Workflow',
+  description: 'A workflow entry',
+  id: 'workflow',
+  attributes: [
+    {
+      type: 'textarea',
+      id: 'comment',
+      name: 'Comment',
+      required: 'save',
+      description: 'Comment required to continue workflow',
+    },
+    {
+      type: 'select',
+      id: 'action',
+      name: 'Action',
+      required: 'save',
+      description: 'Please choose to approve or reject this content.',
+      inputs: {
+        select: {
+          options: [
+            { label: 'Select Action',
```

Comment:
This is a UX issue actually. 

problem:
I need to have the form return either `null`, `approve`, or `reject`. If I don't have an option for `value=""` then there's no-way to have a `select`. I could use radios instead - with no radio selected; or some other idea that might need specific validation because `required` validation only checks for an empty string.

solution:
I was hoping to get this in and have a discussion as to the best way to get this selection (reject or approve) into the form and have it work, without javascript, while at the same time using content-types native validation function.

Could be radios, or a select. Or maybe there are some other idea (should be discussed though)